#! /bin/sh /usr/share/dpatch/dpatch-run
## 80-getweb.in.dpatch by Chris Lamb <chris@chris-lamb.co.uk>

## DP: Fix bashism

@DPATCH@
diff -urNad foo2zjs-20070718dfsg.orig/getweb.in foo2zjs-20070718dfsg/getweb.in
--- foo2zjs-20070718dfsg.orig/getweb.in	2008-05-04 00:13:37.000000000 +0100
+++ foo2zjs-20070718dfsg/getweb.in	2008-05-04 00:14:25.000000000 +0100
@@ -236,7 +236,7 @@
 	tar zxf $HERE/$file
 	cd $HERE
 	echo "The tarball is extracted and the current directory is up to date."
-	echo -e "Remove the tarball (y/n)? \c"
+	printf "Remove the tarball (y/n)? "
 	read ans
 	if [ "$ans" = y ]; then
 	    rm -f $file
